src/main/java/com/hazelcast/aws/AwsClient.java

```diff
@@ -86,17 +86,17 @@ private String resolveIamRole() {
     }
 
     private AwsCredentials prepareCredentials() {
-        if (StringUtil.isNotEmpty(awsConfig.getAccessKey())) {
+        if (StringUtils.isNotEmpty(awsConfig.getAccessKey())) {
             // authenticate using access key and secret key from the configuration
             return AwsCredentials.builder()
                 .setAccessKey(awsConfig.getAccessKey())
                 .setSecretKey(awsConfig.getSecretKey())
                 .build();
         }
 
-        if (StringUtil.isNotEmpty(iamRole)) {
+        if (StringUtils.isNotEmpty(iamRole)) {
             // authenticate using IAM Role
-            LOGGER.info(String.format("Fetching credentials using IAM Role: %s", iamRole));
+            LOGGER.fine(String.format("Fetching credentials using IAM Role: %s", iamRole));
```

Comment:
A small one, but I would keep this log at `info` level if there is no mention of IAM Role usage at any other log. It will display user how we decided from the configuration.

Reply:
Good point. I leave it here as `fine`, because it's logged many times. But I also added an `info` log that is logged once and contains IAM Role attached to EC2 Instance.